agents-for-bedrock/use-case-examples/product-review-agent/agent.py

```diff
@@ -0,0 +1,30 @@
+import boto3
+import uuid
+import logging
+import json
+
+bedrock_agent_runtime_client = boto3.client('bedrock-agent-runtime')
+agent_id = '<your agent id>'
```

Comment:
how is this filled up? i didn't see a documentation telling the user to change this one, did I miss it?